bootstrap.sh

```diff
@@ -448,7 +427,6 @@ install_prerequisites() {
       log "Installing minimal set of requirements"
       export HOMEBREW_NO_AUTO_UPDATE=on
       brew install libxmlsec1 pyenv
-      brew install --cask docker
```

Comment:
Should this be replaced with install steps for colima?

Reply:
sentry's make bootstrap takes care of that with `brew bundle` - added stuff to brewfile here https://github.com/getsentry/sentry/pull/53813